cli/concourseutil.py

```diff
@@ -316,3 +316,20 @@ def trigger_resource_check(
         pipeline_name=pipeline_name,
         resource_name=resource_name,
     )
+
+
+def deploy_or_upgrade_webhook_dispatcher(
+    webhook_dispatcher_name: CliHints.non_empty_string(help="webhook dispatcher config name"),
+    chart_dir: CliHints.existing_dir(help="directory of webhook dispatcher chart"),
```

Comment:
so this must be a directory in which the chart file exists? would it not be more intuitive to specify the helm-chart file instead?

Reply:
yes